admin/src/locales/de.json

```diff
@@ -137,7 +135,9 @@
   },
   "redeemed": "eingelöst",
   "removeNotSelf": "Als Admin/Moderator kannst du dich nicht selber löschen.",
+  "reopen": "Reopen",
```

Comment:
Can we find a German word here? May be just `Öffnen`

Reply:
``neu eröffnen`` · ``wieder eröffnen`` · ``wiedereröffnen``